Libplanet/Net/Swarm.cs

```diff
@@ -33,6 +33,9 @@ public partial class Swarm : ICollection<Peer>, IDisposable
         private static readonly TimeSpan TurnAllocationLifetime =
             TimeSpan.FromSeconds(777);
 
+        private static readonly TimeSpan TurnPermissionLifetime =
+            TimeSpan.FromMinutes(5);
```

Comment:
Is this time interval defined by the specification, or just arbitrary?  If it's arbitrary, it would be good to parametrize this.

Reply:
It was defined in RFC. 

>    The Permission Lifetime MUST be 300 seconds (= 5 minutes).

https://tools.ietf.org/html/rfc5766#section-8

Reply:
Good to link to that in a comment.

Reply:
I've mentioned.